sonar-project.properties

```diff
@@ -3,7 +3,7 @@ sonar.projectKey=smartcontractkit_chainlink-common
 sonar.sources=.
```

Comment:
@chudilka1 as mentioned offline, updated the sonar.projectKey to chainlink-common